epochStart/bootstrap/baseStorageHandler.go

```diff
@@ -37,6 +38,7 @@ type StorageHandlerArgs struct {
 	NodeProcessingMode              common.NodeProcessingMode
 	RepopulateTokensSupplies        bool
 	StateStatsHandler               common.StateStatisticsHandler
+	AdditionalStorageServiceCreator process.AdditionalStorageServiceCreator
 }
 
 func checkNilArgs(args StorageHandlerArgs) error {
```

Comment:
missing nil check for some storage handler args

Reply:
That func is from rc/v1.7.0, not from sovereign, should be fixed in main branch, not  on sovereign